package.json

```diff
@@ -6,7 +6,8 @@
   "dependencies": {
     "serverless-python-requirements": "4.1.1",
     "serverless-plugin-scripts": "1.0.2",
-    "serverless-plugin-datadog": "2.4.0"
+    "serverless-plugin-datadog": "2.4.0",
```

Comment:
Do we want to still have this plugin?

Reply:
Yea, we use it for a couple repos in master right now, need to merge my changes before we can pull this out